addons/sale_timesheet/views/sale_timesheet_portal_templates.xml

```diff
@@ -81,7 +81,7 @@
                 </th>
             </t>
         </xpath>
-        <xpath expr="//thead/tr/th[@t-if='is_uom_day']" position="before">
+        <xpath expr="//th[@id='t_label']" position="before">
```

Comment:
Same here

Reply:
Hello @auon-odoo Implemented the suggested change 
Thank You 😀